Externals/expr/include/expr.h

```diff
@@ -602,6 +629,9 @@ extern "C" {
     else if (src->type == OP_VAR) {
       dst->param.var.value = src->param.var.value;
     }
+    else if (src->type == OP_STRING) {
+      dst->param.str.s = src->param.str.s;
```

Comment:
This will lead to a double free if the expr is copied. You can test this by using expr's macro functionality, which depends on expr_copy. For example, try creating a breakpoint with the following condition: `$(foo, "bar"), foo()`

Instead, this code needs to make a copy of the string:
```
    size_t len = strlen(src->param.str.s);
    dst->param.str.s = (char*)calloc(1, len + 1);
    if (dst->param.str.s != NULL) {
      strncpy(dst->param.str.s, src->param.str.s, len);
    }
```

Reply:
does src need to be checked for NULL at all?

Reply:
It wouldn't hurt to check, I guess. Doesn't seem like it should ever be null in practice.

Reply:
Hmm I think you're right. I'll skip it.